weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -448,7 +485,11 @@ const useAllDynamicColumnNames = (
       tableData.forEach(row => {
         Object.keys(row).forEach(key => {
           const keyAsPath = stringToPath(key);
-          if (isDynamicCallColumn(keyAsPath)) {
+          // Removes summary.usage columns, as they have their own dedicated column.
+          if (
+            isDynamicCallColumn(keyAsPath) &&
+            !key.startsWith('summary.usage')
```

Comment:
nit: i think the `!key.startsWith('summary.usage')` might be better placed as a filter on top of the allDynamicColumns passed into `function buildCallsTableColumns(`... since the implementation of `buildCallsTableColumns` deals with adding derived columns for these fields, it would be nice to co-locate this logic